dex/models/trades/linea/dex_linea_base_trades.sql

```diff
@@ -14,6 +14,7 @@
     , ref('pancakeswap_v3_linea_base_trades')
     , ref('horizondex_linea_base_trades')
     , ref('uniswap_v3_linea_base_trades')
+    , ref('lynex_fusion_linea_base_trades')
```

Comment:
After we add `lynex_fusion_linea_base_trades` into the roll-up spell, we still need to define its schema and tests.

Reply:
alright